app/controllers/explore_controller.py

```diff
@@ -0,0 +1,18 @@
+from app.util.tmdb_helpers import explore
+from flask import jsonify
+
+DESIRED_KEYS = [
+    'title',
+    'overview',
+]
+
+def handle_movie_data(movie):
+    movie_data = { desired_key: movie.get(desired_key) for desired_key in DESIRED_KEYS }
+    movie_data['externalMovieId'] = movie.get('id')
+    movie_data['poster_path'] = 'https://image.tmdb.org/t/p/w370_and_h556_bestv2/{}'.format(movie.get('poster_path'))
```

Comment:
Could you include a comment here pointing to tmdb's documentation on how to construct the url so we don't forget it later?

Reply:
Originally, I just copied the link they use on their own explore page, but after diving deeper it's a bit more complicated. They seem to recommend calling an API to get the correct size enum, but without actually providing any semantic meaning b/w different size labels. See: https://developers.themoviedb.org/3/configuration/get-api-configuration

I think it's probably safe to assume this API won't change in the near future. In the meantime, I'll link to the API page to be able to view image choices.